OCR/frontend/src/componets/FileInput.tsx

```diff
@@ -0,0 +1,246 @@
+import React, {
```

Comment:
had to copy pasta these two components from USDWS bc it kept erroring out when importing